website/docs/crud.md

```diff
@@ -0,0 +1,294 @@
+---
+id: crud
+title: CRUD Handler
+sidebar_label: CRUD Handler
+---
+
+## CRUD
+In `Core-Lib`, we have classes that can handle CRUD  ( Create, Read, Update, Delete ) for us.
+They are as follows
+
+
+
+### CRUD
+This class is used to initialize `CRUD` with database objects, rule validators and database handlers. It is the Base
+class for other classes responsible for CRUD
+```python
+class CRUD(ABC):
+
+    def __init__(self, db_entity, db: SqlAlchemyDataHandlerRegistry, rule_validator: RuleValidator):
+        self._db_entity = db_entity
+        self._db = db
+        self._rule_validator = rule_validator
+```
+`db_entity` is the database class or object to be passed for initialization.
+
+`db` is the `SqlAlchemyDataHandlerRegistry` instance used to connect to the database.

```

Comment:
remove `is the`  
suggestion 

instance of the `SqlAlchemyDataHandlerRegistry` used to connect to the database.